components/expenses/PayExpenseModal.js

```diff
@@ -198,6 +199,27 @@ const getPaymentProcessorFee = (formik, expense, quoteQuery) => {
   }
 };
 
+const calculateAmounts = ({ formik, expense, quote, host, feesPayer }) => {
+  if (formik.values.forceManual) {
+    const totalAmount = { valueInCents: formik.values.totalAmountPaidInHostCurrency, currency: host.currency };
+    const paymentProcessorFee = {
+      valueInCents: formik.values.paymentProcessorFeeInHostCurrency,
+      currency: host.currency,
+    };
+    const VET = expense.currency !== host.currency && totalAmount.valueInCents / expense.amount;
```

Comment:
Shouldn't that be:

```suggestion
    const VET = expense.currency !== host.currency && (totalAmount.valueInCents - paymentProcessorFee) / expense.amount;
```

Reply:
I was trying to convey the effective rate here, do you think it is more useful to display the actual exchange rate?